src/legacy/core_plugins/kibana/public/discover/kibana_services.ts

```diff
@@ -102,3 +100,6 @@ export { Adapters } from 'ui/inspector/types';
 export { Chrome, IInjector } from 'ui/chrome';
 export { registerTimefilterWithGlobalStateFactory } from 'ui/timefilter/setup_router';
 export { FieldName } from 'ui/directives/field_name/field_name';
+export { getFormat } from 'ui/visualize/loader/pipeline_helpers/utilities';
+// @ts-ignore
+export { buildPointSeriesData } from 'ui/agg_response/point_series/point_series';
```

Comment:
Ah, I wasn't aware point series data builder is also used in discover. Unfortunately that one's a big larger than the legacy response handlers (~1k LOC), not sure whether we should duplicate then. @ppisljar IMHO we should move point_series into data together with tabify, wdyt?